resources/js/views/settings/servers/Index.vue

```diff
@@ -186,12 +193,12 @@ export default {
   computed: {
     tableFields () {
       const fields = [
-        { key: 'id', label: this.$t('settings.servers.id'), sortable: true },
-        { key: 'description', label: this.$t('settings.servers.description'), sortable: true },
-        { key: 'status', label: this.$t('settings.servers.status'), sortable: true },
-        { key: 'meeting_count', label: this.$t('settings.servers.meetingCount'), sortable: true },
-        { key: 'participant_count', label: this.$t('settings.servers.participantCount'), sortable: true },
-        { key: 'video_count', label: this.$t('settings.servers.videoCount'), sortable: true }
+        { key: 'id', label: this.$t('settings.servers.id'), sortable: true, tdClass: 'td-max-width-0-lg', thStyle: { width: '8%' } },
+        { key: 'description', label: this.$t('settings.servers.description'), sortable: true, tdClass: 'td-max-width-0-lg' },
+        { key: 'status', label: this.$t('settings.servers.status'), sortable: true, tdClass: 'td-max-width-0-lg', thStyle: { width: '10%' } },
```

Comment:
why do you use the `td-max-width-0-lg` when the text truncate is not used in the cellsl?